src/constructor.io/models/common/SortOptionForModifyRequest.cs

```diff
@@ -0,0 +1,40 @@
+using Newtonsoft.Json;
+
+namespace Constructorio_NET.Models
+{
+    /// <summary>
+    /// Constructor.io Sort Option For Modification Requests.
+    /// </summary>
+    [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
+    public class SortOptionForModifyRequest
```

Comment:
You are absolutely right. I totally forgot that was a thing. I will have to mark the `SortOrder` attribute to be optional on the `SortOption` class though but I'm guessing that should be fine now since we're using the same class to send delta updates.

Reply:
All good, if it seems weird I think we can just keep this class since it's only used internally I believe.

Reply:
No worries, I think having a single `SortOption` class makes things more readable + developer friendly anyways since it's more intuitive to expect the same `SortOption` class to be used but for null values to be ignored. I made the changes but if anyone has a differing opinion, feel free to voice out.

The contrasting examples to this pattern are the different request objects we have, but those act as sort of "recipes" that inform the SDK consumer what can/cannot be passed to a specific function/api call so it makes more sense to have different classes that only contain the attributes specific to those calls.